content/applications/general/payment_acquirers/paypal.rst

```diff
@@ -52,28 +57,37 @@ You can refer to `Paypal Fees <https://www.paypal.com/webapps/mpp/paypal-fees>`_
 Settings in Paypal
 ==================
 
-First, let’s see how to set up your Paypal account in order to build a seamless customer experience
-with Odoo.
+First, set up your Paypal account to build a seamless customer experience with Odoo.
+
+Log into your PayPal account and open the account settings. Then, go to your :menuselection:`account
+menu --> Account settings --> Website payments`.
+
+Enable Auto Return
+------------------
+
+The *Auto Return* feature automatically redirects your customers to Odoo once the payment is
+processed.
 
-Log in and open the settings. Go to :menuselection:`Your account menu --> Account settings -->
-Product & Services --> Website payments` and click **Update** on **Website preferences**.
+From the *Website payments* settings page, go to :menuselection:`Website preferences --> Update -->
+Auto return for website payments` and select **On**. Enter the address of your Odoo database (e.g.,
+`https://yourcompany.odoo.com`) in the **Return URL** field.
 
-Auto Return
------------
+.. note::
+   Any URL will do the job. Odoo only needs the setting to be enabled since it uses another URL.
+
+.. _paypal/enable-pdt:
 
-*Auto Return* automatically redirects your customers to Odoo once the payment is processed. Check
-*Auto Return* and enter your domain name with the suffix ``/shop/confirmation`` as *Return URL*
-(e.g. `https://yourcompany.odoo.com/shop/confirmation`).
+Enable Payment Data Transfer (PDT)
+----------------------------------
 
-This URL is requested in Paypal but not used in practice as Odoo transmits it at each transaction.
-Don’t worry if you manage several sales channels or Odoo databases.
+Enable the *Payment Data Transfer* feature to receive payment confirmations immediately.
 
-Payment Data Transfer (PDT)
----------------------------
+That feature also displays the payment status to the customers and verifies the authenticity of the
```

Comment:
```suggestion
This feature also displays the payment status to the customers and verifies the authenticity of the
```

Reply:
Same paragraph as the line before to have a "why you need this" paragraph followed by a "how you enable this" paragraph?